.github/workflows/isort.yml

```diff
@@ -0,0 +1,16 @@
+name: Run isort
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+      - uses: jamescurtin/isort-action@master
```

Comment:
@mski-iksm I think this workflow should be integrated with Formatter CI (coz this is a kind of "formatter CI")

Reply:
@hirosassa I've moved the isort CI to the bottom of format.yml.

Reply:
Thanks!